packages/gasket-plugin-nextjs/lib/prompt.js

```diff
@@ -49,5 +49,18 @@ module.exports = async function promptHook(gasket, context, { prompt }) {
     newContext.addSitemap = addSitemap;
   }
 
+  if (!('useRedux' in context)) {
```

Comment:
These prompts should be in the redux plugin, not the nextjs plugin, right?

Reply:
Depends.... they could be either place and the impact would if the plugin is in the preset dependencies or not essentially. If we add them to the redux plugin itself then the plugin will need to be a dependency of the preset(s).

Reply:
> If we add them to the redux plugin itself then the plugin will need to be a dependency of the preset(s)
Yes, I think that makes most sense. If it's not, selecting yes if the Nextjs plugin asks about it will have no effect.

Reply:
If a preset does not want redux to be an option, then it just does not include the redux plugin.

Reply:
Moved prompt to the Redux plugin